zrml/court/src/lib.rs

```diff
@@ -20,44 +18,59 @@ mod pallet {
     use frame_support::{
         dispatch::DispatchResult,
         pallet_prelude::StorageMap,
-        traits::{Currency, Hooks, IsType},
+        traits::{Currency, Get, Hooks, IsType, Randomness, ReservableCurrency},
         Blake2_128Concat,
     };
     use frame_system::{ensure_signed, pallet_prelude::OriginFor};
-    use sp_runtime::DispatchError;
+    use sp_runtime::{traits::Saturating, ArithmeticError, DispatchError, SaturatedConversion};
+    use zeitgeist_primitives::{
+        traits::DisputeApi,
+        types::{Market, OutcomeReport, ResolutionCounters},
+    };
     use zrml_market_commons::MarketCommonsPalletApi;
 
+    const MAX_RANDOM_JURORS: usize = 5;
```

Comment:
The maximum capped number of random jurors for a dispute. IMHO, it makes sense the set this value as a constant that can be changes through runtime updates.

Reply:
Yep, this makes sense, although I would set it to a larger number like 13 so that we can have a number of escalations before making it become a chain-wide vote. It could start with a group of 5 jurors, then 7, then 9, 11, 13.

Reply:
This PR is already too big so I opened https://github.com/zeitgeistpm/zeitgeist/issues/232. Such feature should be resolved in the next PR